CMakeLists.txt

```
@@ -3,8 +3,13 @@ project(CMaize VERSION 1.0.0 LANGUAGES NONE)
 
 option(BUILD_TESTING "Should we build and run the unit tests?" OFF)
 
-# Include the fetch module and bring CMakePPLang into scope
-list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
+# If cmake/ directory is not part of the CMAKE_MODULE_PATH, add it so we can
+# find CMaize and CMakePPLang
+list(FIND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}" path_has_cmake)
```

```suggestion
list(FIND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake" path_has_cmake)
```
I think we should check for the exact path we are about to add if it doesn't exist instead of just the parent directory.